src/main/java/com/programmers/springweekly/repository/customer/FileCustomerRepository.java

```diff
@@ -0,0 +1,48 @@
+package com.programmers.springweekly.repository.customer;
+
+import com.programmers.springweekly.domain.customer.Customer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Repository
+public class FileCustomerRepository implements CustomerRepository{
+
+    @Value("${file.customer.path}")
+    private String FILE_PATH;
+    private final Map<UUID, Customer> customerMap = new ConcurrentHashMap<>();
+    private static final Logger logger = LoggerFactory.getLogger(FileCustomerRepository.class);
+
+    @Override
+    public Map<UUID, Customer> getCustomerTypeBlackList() {
+        try {
+            BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(FILE_PATH));
```

Comment:
BufferedReader 같은 스트림을 사용하는 자원은 사용 후 반드시 반환해야합니다.
try-with-resources 문법을 알아보세요~

```suggestion
        try(BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(FILE_PATH))) {
```